todo_app/data/trello_client.py

```diff
@@ -0,0 +1,46 @@
+import requests, os
+from dotenv import load_dotenv
+
+from todo_app.model.Card import Card
+
+TRELLO_TOKEN = os.getenv("TRELLO_TOKEN")
+TRELLO_KEY = os.getenv("TRELLO_KEY")
+TODO_LIST_ID = "6165ed85c7acb11e1c5f4f59"
+DONE_LIST_ID = "6165ed85c7acb11e1c5f4f5b"
+
+# API Client for Trello Board, see UsingTrello.md 
+# and Trello API Docs: https://developer.atlassian.com/cloud/trello/rest/api-group-cards/#api-cards-id-get
+
+def create_todo_url(name):
+    return "https://api.trello.com/1/cards?idList=6165ed85c7acb11e1c5f4f59&key="+TRELLO_KEY+"&token="+TRELLO_TOKEN+"&name="+name
```

Comment:
And this embedded list id should be switched to use the relevant variable, otherwise it's very easy for someone to miss this (spoken from experience!) and have the app break when this is used